src/Observers/NovaInlineRelationshipObserver.php

```diff
@@ -94,7 +95,7 @@ public function getRelationshipObserver(Model $model, $relationship): Relationsh
      */
     protected function getModelRelationships(Model $model)
     {
-        return collect(Nova::newResourceFromModel($model)->fields(request()))
+        return collect(Nova::newResourceFromModel($model)->fields(Nova::createFrom(request())))
```

Comment:
This should be NovaRequest instead of Nova

Reply:
I am getting the same issue with request(). Are we planning to merge the pull request that fixes the issue?